### ISSUE TITLE
Fix dynamodb setup documentation

### ISSUE THREAD

**Issue by ryan-lane** [lyft/confidant#44]
The table definitions for dynamodb in the docs don't match up well with what AWS has in the console when you create tables and indexes. We should use similar info and we should show screenshots (and/or a screencast) of the process.

**Comment by AndreyKrasnov** [comment]
Hello

Could you show settings screenshots of DynamoDB table?
I do not quite understand how DynamoDB is configured according to the documentation.

Thanks

**Comment by ryan-lane** [comment]
Yeah. I think someone was working on this, but I can add some screenshots if they don't. I can also make it possible for Confidant itself to generate the tables. It already supports it, but it only works when a dynamo url is specified (for local dynamo). I can make a config option that'll make it work, though.

**Comment by ryan-lane** [comment]
Though not a fix for the documentation, I added the ability (in branch 1.1) for Confidant to generate its own table: https://github.com/lyft/confidant/issues/56

**Comment by ambrons** [comment]
I updated to `branch 1.1`, built a new docker and deployed.  I was sure to make the ec2 container role have dynamodb create table privelages.  There were no errors related to not having permissions however it did *not* create the table for me.

I looked back at your documentation and was able to determine what was suppose to be configured.

*Primary partition key:* `id` [String]
Read/Writes: 10/10
Additional Index Keys:
  `data_type_date_index`:
    *Primary Key:* `data_type` [String]
    *[check] Add sort key:* `modified_date` [String]
    *Read/writes:* 10/10
  `data_type_revision_index`:
    *Primary Key:* `data_type` [String]
    *[check] Add sort key:* `revision` [Number]
    *Read/writes:* 10/10

**Comment by ryan-lane** [comment]
Whoops. Sorry, I forgot to mention that there's a config option that controls this: DYNAMODB_CREATE_TABLE

I _thought_ I had added this to 1.0, but it seems it's only in 1.1. I've opened an issue to backport this: https://github.com/lyft/confidant/issues/66
